src/server.js

```diff
@@ -378,8 +377,32 @@ function initializeClient(socket, client, token, lastMessage) {
 		client.unregisterPushSubscription(token);
 	});
 
-	socket.on("sign-out", () => {
-		delete client.config.sessions[token];
+	const sendSessionList = () => {
+		const sessions = _.map(client.config.sessions, (session, sessionToken) => ({
+			current: sessionToken === token,
+			active: typeof _.find(client.attachedClients, (u) => u.token === sessionToken) !== "undefined",
```

Comment:
I know the whole reasoning behind that `typeof ... !== "undefined"` but it is frankly overrated, `... !== undefined` is perfectly fine (I read an article not long ago that I can't find, which was dismantling this pretty well; in particular the author was saying than no one ever applies this amount of strictness outside of this case).

In fact, we use `===/!== undefined` in many places already :sweat_smile: